### ISSUE TITLE
New messages are not returned when column has unique validator

### ISSUE THREAD

**Issue by Jank1310** [Jank1310/datadolphin#66]
Expected: Return new messages for cell if column has unique validator.

**Comment by Jank1310** [comment]
@Flofie Wäre es nicht einfacher im Updatehandler eine revalidate Funktion aufzurufen und dieser sowas wie changes: {rowId:string, columnId:string}[] zu übergeben? Dann kann diese Funktion prüfen, wie neu validiert werden sollte und auch die new Messages für die changes zurückgeben.

**Comment by Flofie** [comment]
We need to better refine this. This is what we agreed on. If there is a unique validator, then chnageCoumn is returned (not newMessages) and the whole page is reloaded.

**Comment by Jank1310** [comment]
Yes, but the data is reloaded in background and in the meantime it is not possible to update the messages of the edit cell to get a quicker feedback for the new value. I added a workaround for now :) #67
